scripts/export_data.sh

```
@@ -0,0 +1,35 @@
+
+TMPFILE=$(mktemp)
+trap "rm -f $TMPFILE" EXIT
+
+echo "Starting export with API key: $API_KEY"
+
+QUERY="SELECT * FROM flaws WHERE status = 'active'"
+eval "python3 -c \"from osidb_bindings.export import export_to_format; print('done')\" $EXPORT_FORMAT"
+
+if [ -n "$EXPORT_FORMAT" ]; then
+    python3 -m osidb_bindings.export \
+        --format $EXPORT_FORMAT \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)